src/xrCore/xrDebug.cpp

```diff
@@ -682,7 +681,7 @@ static void invalid_parameter_handler(
         line = __LINE__;
         xr_strcpy(mbFile, __FILE__);
     }
-    xrDebug::Fail(ignoreAlways, {mbFile, int(line), mbFunction}, mbExpression, "invalid parameter");
+    xrDebug::Fail(ignoreAlways, { mbFile, int(line), mbFunction }, mbExpression, "invalid parameter");
```

Comment:
Why?

Reply:
You find this no manually, didn't you? If you've been having some linter, why doesn't documentation have link to it? This would make life easier for everyone.

Reply:
For now we only have a Clang-Format config that's ready for use

Reply:
I look at diff and every single change shows up right there. Every change is made with a reason behind it, right? I guess there's no reason here because this change was made by VS which automatically reformatted pasted code. That is why you should check the diff before committing.

Reply:
Coding conventions page in wiki doesn't explicitly require that, but there's an example marked as `GOOD` that shows initialization formatting:

` Size size = {1024, 16};`

Reply:
>I look at diff and every single change shows up right there. Every change is made with a reason behind it, right?
I like your approach